src/audacity.desktop.in

```diff
@@ -22,6 +27,7 @@ GenericName[tr]=Ses Düzenleyici
 GenericName[uk]=Редактор звукових файлів
 GenericName[zh_CN]=音频编辑器
 GenericName[zh_TW]=音訊編輯器
+# comments added 
```

Comment:
Also, you can leave out these comments. I can see from git which lines you added.